.travis.yml

```diff
@@ -26,6 +26,3 @@ notifications:
   email:
     recipients:
       - kiran.mova@cloudbyte.com
```

Comment:
Can you also update the above email id to include all the maintainers of this repo please.

Reply:
done